evap/evaluation/models.py

```diff
@@ -916,14 +971,31 @@ def send_to_user(self, user, courses=None, cc=True):
         else:
             cc_addresses = []
 
+        grade_documents = len(grade_document_courses) > 0
+        evaluation_results = len(evaluation_results_courses) > 0
+
         mail = EmailMessage(
-            subject = self.render_string(self.subject, {'user': user, 'courses': courses}),
-            body = self.render_string(self.body, {'user': user, 'courses': courses}),
+            subject = self.render_string(self.subject, {
+                'grade_documents': grade_documents,
+                'evaluation_results': evaluation_results
+            }),
+            body = self.render_string(self.body, {
+                'user': user,
+                'courses': courses,
+                'grade_documents': grade_documents,
+                'evaluation_results': evaluation_results,
+                'grade_document_courses': grade_document_courses,
+                'evaluation_results_courses': evaluation_results_courses
+            }),
             to = [user.email],
             cc = cc_addresses,
             bcc = [a[1] for a in settings.MANAGERS],
             headers = {'Reply-To': settings.REPLY_TO_EMAIL})
-        mail.send(False)
+        
+        try:
+            mail.send(False)
+        except Exception:
+            logging.getLogger(__name__).exception(('An error occured when sending email "{}" to {}.').format(subject, user.username))
```

Comment:
the body might be quite long and there might be quite a lot of failing messages if something goes wrong. should we put that much text into the log file?